packages/cli/src/lib/quire/index.js

```diff
@@ -184,6 +183,74 @@ async function install(version, options={}) {
   await execaCommand('npm install --save-dev')
 }
 
+/**
+ * Install a specific version of `quire-11ty` directly into a quire project
+ *
+ * @param  {String}  projectPath  Absolute system path to the project root
+ * @param  {String}  version  Quire-11ty semantic version
+ * @param  {Object}  options  options passed from `quire new` command
+ * @return  {Promise}
+ */
+async function installInProject(projectPath, version, options={}) {
+  console.debug(`[CLI:quire] installing quire-11ty@${version} into ${projectPath}`)
+
+  /**
+   * delete `package.json` from starter project, as it will be replaced with
+   * `package.json` from `@thegetty/quire-11ty`
+   */
+  await git
+    .cwd(projectPath)
+    .rm(['package.json'])
+    .catch((error) => console.error('[CLI:error] ', error))
+
+  const temp11tyDirectory = '.temp'
+  /**
+   * `Destination` is relative to `node_modules` of the working-directory
+   * so we have included a relative path to parent directory in order to
+   * install versions to a different local path.
+   * @see https://github.com/scott-lin/install-npm-version
+   */
+  const installOptions = {
+    Destination: path.join('..', temp11tyDirectory),
+    Debug: false,
+    Overwrite: options.force || options.overwrite || false,
+    Verbosity: options.debug ? 'Debug' : 'Silent',
+    WorkingDirectory: projectPath
+  }
+  await inv.Install(`${PACKAGE_NAME}@${version}`, installOptions)
+
+  // delete empty `node_modules` directory that `install-npm-version` creates
+  const invNodeModulesDir = path.join(projectPath, 'node_modules')
+  if (fs.existsSync(invNodeModulesDir)) fs.rmdir(invNodeModulesDir)
+
+  // Copy all files installed in `.temp` to projectPath
```

Comment:
Init starter also initializes a new package config (`package.json`), will this overwrite that file?